lib/components/UploadList.jsx

```diff
@@ -30,6 +30,27 @@ var UploadList = React.createClass({
     groupsDropdown: React.PropTypes.bool.isRequired
   },
 
+  renderErrors: function() {
+    //do any of the target uploads have errors
+    var uploadsWithError = _.filter(this.props.targetedUploads, function(upload) {
+      return _.isEmpty(upload.error) === false;
+    });
+
+    if(_.isEmpty(uploadsWithError)) {
+      return null;
+    }
+
+    return _.map(uploadsWithError, function(upload){
+      return (
+        <div key={upload.key} className="UploadList-error-item">
+          <div className="UploadList-error-name">{upload.name}</div>
```

Comment:
Maybe this section should have a headline at the top. "Upload Errors" or something? Either that or maybe make this "Upload error in " + upload.name